packages/v4/patternfly-docs/content/design-guidelines/components/menu/menu.md

```diff
@@ -97,6 +97,36 @@ When using red text for destructive actions, it is still recommended to require
 
 <img src="./img/red-delete-text.png" alt="drilldown menu" width="935"/>
 
+## Disabling menus and menu options
+Menus or menu items can be disabled for multiple reasons. The reason an action is disabled informs which design pattern you should use.
+
+### **An action is unavailable due to an unmet prerequisite, condition, permission, or status.** 
+If a user needs to complete a prerequisite to enable an action, disable the action and add a [tooltip](components/tooltip). The tooltip should explain what the user needs to do to enable the action. 
+
+<img src="./img/dropdown-disabled-action-tooltip.png" alt="disabled action tooltip" width="935"/>
+
+Example: A user can’t perform bulk actions until they select resources in the list.
+
+Example: A user with view-only privileges looks at a Pod’s details page and the options within the Actions dropdown are disabled.
+
+If an action is dependent on an unmet condition or status in the console, disable the action with a [tooltip](components/tooltip) explaining why the action currently is unavailable.
```

Comment:
Feel like this sentence should be deleted/merge with the first sentence in this section because they're basically saying the same thing (?)